index.js

```diff
@@ -0,0 +1,172 @@
+import { API_KEY } from "./config.js";
+
+const input = document.getElementById("myInput");
+const btn = document.getElementById("myBtn");
+const currentLoc = document.getElementById("currentLoc");
+const forecast = document.getElementById("forecastContainer");
+const loader = document.getElementById("loading");
+
+function displayLoading() {
+  loader.classList.add("display");
+
+  setTimeout(() => {
+    loader.classList.remove("display");
+  }, 5000);
+}
+
+function hideLoading() {
+  loader.classList.remove("display");
+}
+
+async function getDefault() {
+  let storedCity = localStorage.getItem("lastcity");
+  let myCity = JSON.parse(storedCity);
+
+  renderData(myCity);
+}
+
+async function getCity(city) {
+  if (!city) {
+    alert("Please enter a city name");
+
+    return;
+  }
+  const url = `https://api.openweathermap.org/data/2.5/weather?q=${city}&appid=${API_KEY}`;
+
+  const options = {
+    method: "GET",
+    headers: {},
+  };
+
+  try {
+    const response = await fetch(url, options);
+    const result = await response.json();
+
+    if (result.coord === undefined) {
+      alert("You are looking for a not existing city");
+      return;
+    }
+
+    return {
+      lat: result.coord.lat,
+      lon: result.coord.lon,
+    };
+  } catch (error) {
+    console.error(error);
+  }
+}
+
+async function getWeather(coords) {
+  if (coords === undefined) {
```

Comment:
You can simplify this "if" statement by writing if (!coord), this way you would be checking if it is null, undefined or false